test/integration/basic_monitor_crossconnect_metrics_test.go

```diff
@@ -21,21 +24,26 @@ func TestSimpleMetrics(t *testing.T) {
 		t.Skip("Skip, please run without -short")
 		return
 	}
-	if !nsmd_test_utils.IsBrokeTestsEnabled() {
-		t.Skip("Skipped for a while, will be enabled soon")
-		return
-	}
 	k8s, err := kube_testing.NewK8s(true)
 	Expect(err).To(BeNil())
 
 	defer k8s.Cleanup()
 
 	nodesCount := 2
+	requestPeriod := time.Second
+	variables := map[string]string{
+		vppagent.DataplaneMetricsCollectorEnabled:       "true",
+		vppagent.DataplaneMetricsCollectorRequestPeriod: requestPeriod.String(),
+	}
 
-	nodes := nsmd_test_utils.SetupNodes(k8s, nodesCount, defaultTimeout)
-	nsmd_test_utils.DeployICMP(k8s, nodes[nodesCount-1].Node, "icmp-responder-nse-1", defaultTimeout)
-	nsc := nsmd_test_utils.DeployNSC(k8s, nodes[0].Node, "nsc1", defaultTimeout)
+	nodes := nsmd_test_utils.SetupNodesConfig(k8s, nodesCount, defaultTimeout, []*pods.NSMgrPodConfig{
```

Comment:
Please also use default values for NSmgr.

Reply:
thanks, fixed